Automattic/sensei
# Fix student "Message Reply Received" email not sending from lesson or quiz page

Resolves #6647.

## Proposed Changes
Validate the post type when determining whether to send the email, as the original message may have been sent from a course, lesson or quiz page.

## Testing Instructions
1. Enable the `email_customization` feature flag.
2. Enable the "Message Reply Received" student email.
3. As a student, contact the teacher from a course, lesson and quiz page. To send from a quiz page, you'll need to submit the quiz first. You should then see the Contact Teacher button in Learning Mode:

![Screenshot 2023-03-21 at 2 27 29 PM](https://user-images.githubusercontent.com/1190420/226706529-28b5cc7e-3b22-4fe1-85b3-6429284ead24.jpg)
4. As the teacher of that course, find the messages on the `/messages` page and reply to each of the student's messages.
5. Ensure 1 email is sent for each email reply.

## Pre-Merge Checklist
